tests/zero_code_change/tensorflow2_gradtape_integration_tests.py

```diff
@@ -0,0 +1,169 @@
+"""
+WARNING: This test is useful for DLC testing.
+
+Test with Keras GradientTape.
+
+We check that certain tensors are saved.
+Here in the test suite we delete the hook after every script.
+"""
+# Standard Library
+import argparse
+
+# Third Party
+import tensorflow.compat.v2 as tf
+
+# First Party
+import smdebug.tensorflow as smd
+from smdebug.core.utils import SagemakerSimulator
+
+
```

Comment:
Disable progress bar tfds progress bar?

Reply:
ok, let me try that

Reply:
reducing number of epochs to 3. On GPU.

`time timeout 600s python mock/tests/zero_code_change/tensorflow2_gradtape_integration_tests.py`
real    2m48.834s
user    2m58.264s
sys     0m9.412s

`time timeout 600s python mock/tests/zero_code_change/tensorflow2_integration_tests.py`
real    6m3.416s
user    23m54.660s
sys     103m16.680s


tfds.disable_progress_bar may not be relevant in these tests as tfds is not used. probably changing verbose level in fit and evaluate will help. should i try that?